contrib/upload-kktix-ticket-csv-to-bigquery.py

```diff
@@ -322,5 +322,61 @@ def test_hash_email(self):
         )
 
 
+class Test2019Ticket(unittest.TestCase):
+    """python -m unittest upload-kktix-ticket-csv-to-bigquery.py"""
+
+    CANONICAL_COLUMN_NAMES_2019 = [
+        "ticket_type",
+        "payment_status",
+        "tags",
+        "paid_date",
+        "price",
+        "invoice_policy",
+        "invoiced_company_name",
+        "unified_business_no",
+        "dietary_habit",
+        "need_shuttle_bus_service",
+        "size_of_tshirt_t",
+        "years_of_using_python",
+        "area_of_interest",
+        "organization",
+        "job_title",
+        "country_or_region",
+        "gender",
+        "email",
+    ]
+
+    @classmethod
+    def setUpClass(cls):
+        cls.df = pd.read_csv("./data/corporate-attendees-2019.csv")
+        cls.sanitized_df = sanitize_column_names(cls.df)
+
+    def test_column_number(self):
+        self.assertEqual(len(self.sanitized_df.columns), 18)
+
+    def test_column_title_content(self):
+        for column in self.sanitized_df.columns:
+            self.assertTrue(column in self.CANONICAL_COLUMN_NAMES_2019)
```

Comment:
```suggestion
            self.assertIn(column, self.CANONICAL_COLUMN_NAMES_2019)
```

Reply:
Good catch (again)! Thank you for your wonderful heads-up!